### PR TITLE
Bump some dependencies

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

**Comment by lnicola** [comment]
Ah, I didn't get the notification for the r+. The latest commit upgrades some npm packages and eslint complains a little about the TypeScript version. Do we care about that?

**Comment by lnicola** [comment]
Actually, I think it was complaining even before my changes (we were already on 3.9).

**Comment by lnicola** [comment]
Rebased.

**Comment by matklad** [comment]
bors d+

**Comment by lnicola** [comment]
bors r+

...I guess

**Comment by lnicola** [comment]
Oof...

@matklad r=matklad

**Comment by matklad** [comment]
Build succeeded:
  * [Rust (macos-latest)](https://github.com/rust-analyzer/rust-analyzer/runs/703815667)
  * [Rust (ubuntu-latest)](https://github.com/rust-analyzer/rust-analyzer/runs/703815656)
  * [Rust (windows-latest)](https://github.com/rust-analyzer/rust-analyzer/runs/703815660)
  * [TypeScript](https://github.com/rust-analyzer/rust-analyzer/runs/703815637)

File: editors/code/package.json [hunk 1]
```diff
@@ -39,21 +39,21 @@
...
     },
     "devDependencies": {
-        "@rollup/plugin-commonjs": "^11.1.0",
-        "@rollup/plugin-node-resolve": "^7.1.3",
+        "@rollup/plugin-commonjs": "^12.0.0",
+        "@rollup/plugin-node-resolve": "^8.0.0",
         "@types/glob": "^7.1.1",
         "@types/mocha": "^7.0.2",
-        "@types/node": "^12.12.39",
+        "@types/node": "^14.0.5",
```
Comment (Veetaha) [inline editors/code/package.json:46]: I've just noticed, that this is not what we want to do. Vscode is using nodejs12, @lnicola I recommend leaving it under v12 major to keep us safe

**Pull request merged**